src/DetectionDetail.cs

```diff
@@ -31,6 +31,9 @@ public class DetectionDetail
                     .Split('(').Last()
                     .Split(')').First()
                     .Trim();
+#if NETCOREAPP3_0
+                Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);

```

Comment:
I don't this this is the correct place? This is the domain class?

Reply:
Sorry, but I'm not good enough to understand what you asked.

I can say that it is written in the [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.text.codepagesencodingprovider.instance?view=netcore-3.0)

Reply:
I would expect this in UTF-unknown/src/CharsetDetector.cs

Reply:
it looks better